src/components/Search/Search.js

```diff
@@ -1,12 +1,13 @@
 
 function Search(props) {
 
+
   return (
     <section className="search">
       <h1 className="search__title">{props.searchTitle}</h1>
       <p className="search__subtitle">{props.searchSubtitle}</p>
       <form name="query" id="search-form" className="search__form">
```

Comment:
you need to control forms in `React` way. To read more about `controlled components`: https://reactjs.org/docs/forms.html#controlled-components